pkg/cortex/cortex.go

```diff
@@ -249,6 +254,7 @@ func (t *Cortex) Stop() error {
 	for i := len(deps) - 1; i >= 0; i-- {
 		t.stopModule(deps[i])
 	}
+	t.server.Stop()
```

Comment:
Shouldn't it be called in `stopModule` above?

Reply:
The server module only creates server object and initiates the shutdown. The server is being started and stopped manually in the cortex module. Maybe it should be moved before we call any stopModule.

Reply:
Looks like it should. Will remove it.